launcher

```diff
@@ -413,6 +414,13 @@ else
   restart_policy=${restart_policy:---restart=always}
 fi
 
+run_ruby_script() {
+  ruby_file=$1
+  arguments=$2
+  cp containers/"$config".yml ruby_scripts/config.yml  # Add config to the data volume for easy access
+  echo `$docker_path run $user_args --rm -i -a stdout -a stdin -v /var/discourse/ruby_scripts:/ruby_scripts $image ruby /ruby_scripts/"$ruby_file" "$arguments"`
```

Comment:
this is forcing launcher to be installed in /var/discourse ... can not have this tight dependency, use relative to pwd paths